projects/cdk/services/hovered.service.ts

```diff
@@ -25,7 +25,7 @@ export class TuiHoveredService {
     ) {
         this.documentEvents$ = merge(
             typedFromEvent(documentRef, 'mousemove'),
-            typedFromEvent(documentRef, 'touchend'),
+            typedFromEvent(documentRef, 'touchstart', {capture: true}),
```

Comment:
Why do we need capture?

Reply:
Otherwise if we touch one element then second, event for the second element starts before the first event ends.

Reply:
can we write tests for these cases?

Reply:
test added